### PR TITLE
Add Tetris Competition Winner to MM 2024 website

### PR DESCRIPTION
**Pull request by dannyl1u**

### CODE REVIEW

File: public/events/2024/mm/index.html [hunk 1]
```diff
@@ -311,6 +311,14 @@ <h3>Most Entertaining: Moving Mountains</h3>
         </a>
       </p>
       <br />
+      <h3>Tetris Competition Winner: Open-WebUI</h3>
+      <p>
+        By <i>Open-WebUI</i><br />
```
Comment (micahdbak) [inline public/events/2024/mm/index.html:316]: for here, please do "Danny Liu (Contributor to Open-WebUI)"

And modifier "See Project" to "Shameless Plug"

If you make these changes I'll merge! :DD

**Review by micahdbak — COMMENTED** [review]
Small changes, then I'll approve and merge :)

File: public/events/2024/mm/index.html (hunk 1 above)
Comment (dannyl1u) [reply public/events/2024/mm/index.html]: Updated in latest commit :)

**Review by micahdbak — APPROVED** [review]
Lovely!

**Pull request merged**